lib/resqued/master_state.rb

```diff
@@ -1,11 +1,14 @@
 module Resqued
+  # Tracks state from the master process. On re-exec, this object will
+  # be serialized and passed to the new master.
   class MasterState
     def initialize
       @listeners_created = 0
       @listener_states = {}
     end
 
-    # Public: When starting fresh, from command-line options, assign the initial values.
+    # Public: When starting fresh, from command-line options, assign the
+    # initial values.
     def init(options)
       @config_paths = options.fetch(:config_paths)
       @exec_on_hup  = options.fetch(:exec_on_hup, false)
```

Comment:
Good catch! I am going to leave it in place for now so that if we end up rolling back in prod, we'll preserve the setting in the previous version of resqued. But I'll follow up later to remove this setting.